src/Avalonia.Controls/Button.cs

```diff
@@ -78,9 +79,17 @@ public class Button : ContentControl, ICommandSource
         public static readonly StyledProperty<bool> IsPressedProperty =
             AvaloniaProperty.Register<Button, bool>(nameof(IsPressed));
 
+        /// <summary>
+        /// Defines the <see cref="Flyout"/> property
+        /// </summary>
+        public static readonly DirectProperty<Button, FlyoutBase> FlyoutProperty =
```

Comment:
ContextMenu and ContextFlyout are StyledProperty, so it's possible to set them from styles and have multiple prioritized values.
Why Button.Flyout is simple DirectProperty?

Reply:
What if flyout was changed while it was opened? Not only on button, but also ContextFlyout one too.
I don't know actually, how it handled in UWP now.

Reply:
Changed to StyledProperty